### DISCUSSION TITLE
- Thank you -

### DISCUSSION THREAD

**Discussion by bbernhard1** [immich-app/immich#12397] [General]
Hi Immich team
I guess it is time to send you a big <THANK YOU> 
Improvements during the past few revisions are really great and cover everything I missed when I joined Immich few month ago.
At the beginning I was a bit skeptic because major features (folder view, tags ...) were missing. Only the outstanding SmartSearch kept in standby. So I decided to observe the progress of the project for a while. 
Progress was great, so I decided today to go all in and shut down my former gallery software. Now everything is handover to the couple of Immich + ACDSEE.
As respect to you work I ordered a full server license few days ago.

So, thank you again and nice greats 
Bernhard from Villach/Austria/Europe

**Comment by alextran1502** [comment]
Thank you for your kind words! I hope you will enjoy the application more and more
